acceptancetests/assess

```diff
@@ -252,11 +256,14 @@ def parse_args():
     pass_through.add_argument('--large', '--large-test-enabled', action='store_true', dest="large_test_enabled",
                               help="Use a large file for testing. (Relevant only with resources test)")
     pass_through.add_argument('--agent-timeout', metavar="S", type=int, default=1800,
-                              help='The time to wait for agents to start (Relevant only with resources test) [default: 1800]')
+                              help='The time to wait for agents to start (Relevant only with resources test) ['
```

Comment:
Nit on readability: I would break these earlier to keep the braced expressions together.